akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -98,6 +101,15 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
               }
             case _ ⇒
           }
+
+          // once captured, optional verbose logging of event
+          e match {
+            case _: SeenChanged ⇒ // ignore
+            case event ⇒
+              if (cluster.settings.LogInfoVerbose)
+                logInfo("Cluster Node [{}] dc [{}] - event {}", selfAddress, selfDataCenter, event)
```

Comment:
I think the `cluster.InfoLogger` is already taking care of the "Cluster Node [{}] dc [{}]" prefix, so this should be
```
logInfo(" - event {}", event)
```

Reply:
What if they do not have log-info = on and just verbose = on? I didn't want to assume or require the basic info is on.

Reply:
but isn't the `logInfo` method imported via above `import cluster.InfoLogger._`?

I think it's fine to enable `LogInfo` when `LogInfoVerbose` is enabled.

Reply:
@patriknw @chbatey suggestions made. I like @raboof's idea, perhaps a new ticket...